src/com/teamcautionrobotics/robot2018/Robot.java

```diff
@@ -31,7 +31,13 @@
     Gamepad manipulator;
 
     Intake intake;
-    Climb climb;
+    Tower tower;
+
+    boolean towerRaiseStatePressed = false;
```

Comment:
These are rather clunky names. Something more like `liftRaiseButtonPressed` and `liftLowerButtonPressed would be better.